src/dynamics/rigid_body.rs

```diff
@@ -115,8 +119,18 @@ impl RigidBodyHandle {
         })
     }
 
+    /// Override velocity vector.
+    pub fn reset_linear_velocity<T>(&mut self, vel: T)
```

Comment:
Most functions did not use abbreviations for variable names, like ```velocity```  instead of ```vel```. 
It's not that important, but let's try to follow the same naming policy at least for public api.